examples/otel-demo/cleanup.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# OpenSearch Observability Stack Cleanup Script
+# This script removes all components deployed by the deployment script
+
+# Colors for output
+RED='\033[0;31m'
```

Comment:
is there some AI assistant that creates this nonsense? I keep seeing it across many PRs.

Sure, colored logging is neat, but having the same functions redefined in every script is ridiculous. And the same can be achieved with plain `echo` with occasional emojis when attention needs to be called to something like an error.

```
echo "🛑 error message"
```